xptifw/CMakeLists.txt

```diff
@@ -1,6 +1,6 @@
-cmake_minimum_required(VERSION 3.20.0)
+cmake_minimum_required(VERSION 3.5)
```

Comment:
why is it 3.5? is it intentional change?

Reply:
I was testing with CMake 4.0 and that was the minimum requirement - reverted back to 3.20.0